doc/classes/OS.xml

```diff
@@ -430,6 +438,18 @@
 				Not to be confused with [method get_data_dir], which returns the [i]global[/i] (non-project-specific) user home directory.
 			</description>
 		</method>
+		<method name="get_version" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the OS' actual technical version number and not its user-friendly version used in marketing. This helps distinguishing between minor versions or even insider and custom builds.
+				For Windows, the major and minor version plus build number are returned. For example [code]10.0.9926[/code] instead of [code]Windows 10[/code], and [code]6.0.x[/code] instead of the marketed version [code]Vista[/code].
+				For rolling distributions, such as Arch Linux, it returns an empty String.
+				For macOS and iOS, the major and minor version plus patch number are returned.
+				For UWP, the device family version is returned.
+				For Android, the SDK version and incremental build number are returned.
+				Web is not supported.
```

Comment:
```suggestion
				Returns the exact production and build version of the operating system. This is different from the branded version used in marketing. This helps to distinguish between different releases of operating systems, including minor versions, and insider and custom builds.
				For Windows, the major and minor version are returned, as well as the build number. For example,  the returned string can look like [code]10.0.9926[/code] for a build of Windows 10, and it can look like [code]6.0.6000[/code] for a build of Windows Vista. For rolling distributions, such as Arch Linux, an empty string is returned. For macOS and iOS, the major and minor version are returns, as well as the patch number. For UWP, the device family version is returned. For Android, the SDK version and the incremental build number are returned.
				[b]Note:[/b] This method is not supported on the web platform.
```

Same consideration.